### PR TITLE
🐛 Source Bing Ads - Fix Campaigns stream misses Audience and Shopping

### LINKED ISSUE #17252: Source BingAds - Campaigns stream misses Audience and Shopping campaigns
## Environment
- **Source Connector and version**: Bing Ads `0.1.12` 
- **Step where error happened**: After sync job (which is successful), when data modelling for report 

## Current Behavior
While `campaign_performance_report_daily` stream has data for all campaign types (Search, Audience, Shopping), `campaigns` only retrieves Search campaigns type. So the `campaigns` stream is technically incomplete and, for example, we can't get the CampaignType in campaign reports with a simple join.

## Expected Behavior
The `campaigns` stream should include Search, Audience and Shopping campaigns.

## Additional information
- According to the [Bing API docs](https://learn.microsoft.com/en-us/advertising/campaign-management-service/getcampaignsbyaccountid?view=bingads-13#request-body), the `GetCampaignsByAccountId` service only retrieves Search campaigns by default. To get Shopping and Audience campaigns, we need to explicit them in the request
- Change should probably occur [here](https://github.com/airbytehq/airbyte/blob/9dacd5df4dbb49c4fb473b6dd8a1330fdeb23ddc/airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py#L205)

## Steps to Reproduce
1. Run a successful job for `campaigns` and `campaign_performance_report_daily` streams
2. If you look at distinct CampaignType in `campaigns`, you only get "Search"
3. If you try to do a left join (main table is `campaign_performance_report_daily` and joined table is `campaigns`) to get CampaignType in the reports then you'll have NULL for campaigns that are not "Search" campaigns

## Are you willing to submit a PR?
Not right away but probably in the future!

### PR DESCRIPTION
**Pull request by rach-r**
## What
Today, only Search campaign type is fetched in Campaigns stream, while all campaigns are available in campaign reports.
This PR aims to fix this by explicitly calling all campaign types in request parameters.

Closes [this issue](https://github.com/airbytehq/airbyte/issues/17252).

## How
By explicitly calling all campaign types in request parameters.
Here is the [API documentation](https://learn.microsoft.com/en-us/advertising/campaign-management-service/getcampaignsbyaccountid?view=bingads-13#request-body) mentioning how it should be retrieved.

## Pre-merge Checklist

<summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Documentation updated
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by alafanechere — COMMENTED** [review]
Thank you for this initiative! 
I'll run the acceptance tests on our sandbox account below.

**Comment by alafanechere** [comment]
Requesting review from @grubberr / @bazarnov for awareness as this connector is GA.

**Review by alafanechere — APPROVED** [review]
👍 Tests are passing. @rach-r feel free to make the minor suggestion I shared above.

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py [hunk 1]
```diff
@@ -207,6 +207,9 @@ class Campaigns(BingAdsStream):
         " DynamicFeedSetting MaxConversionValueBiddingScheme MultimediaAdsBidAdjustment"
         " TargetImpressionShareBiddingScheme TargetSetting VerifiedTrackingSetting"
     )
+    campaign_types: str = (
```
[the remark this answers is not in the document]
Comment (bazarnov) [reply airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py:210]: +1

**Review by bazarnov — APPROVED** [review]
LGTM, but make sure this comment is processed: https://github.com/airbytehq/airbyte/pull/17873/files#r993262907

**Review by bazarnov — APPROVED** [review]

**Comment by alafanechere** [comment]
Thanks @rach-r for the contribution! 🎉 Feel free to open more PR like this one ;-)

**Comment by lluisgassovillarejo** [comment]
Hi,
**We have found this campaign-type definition problem also in the Cloud version**.

The problem summarizes that we are making a move from Fivetran to Airbyte Cloud and:
- Fivetran has three campaigns: Shopping, Search, and Audience.
- On Airbyte, only one appears: Search.

So we find correct but incomplete data.
What can we do to solve this problem?

A greeting and thanks for the help.

**Pull request merged**